src/Service.Tests/SqlTests/GraphQLCollationTests/GraphQLCollationTestBase.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Azure.DataApiBuilder.Service.Tests.SqlTests.GraphQLCollationTests
+{
+    [TestClass]
+    public class GraphQLCollationTestBase : SqlTestBase
+    {
+        #region Tests
+
+        /// <summary>
+        /// Compares SQL Database Query with GraphQL Query
+        /// </summary>
+        public async Task CapitalizationResultQuery(string type, string item, string dbQuery)
```

Comment:
```suggestion
        public async Task CapitalizationResultQuery(string objectType, string fieldName, string dbQuery)
```

more descriptive var names that align with GraphQL nomenclature per https://spec.graphql.org/October2021/#sec-Objects